libjob_queue/src/lsf_driver.c

```diff
@@ -919,12 +923,16 @@ void * lsf_driver_submit_job(void * __driver ,
       lsf_submit_method_enum submit_method = driver->submit_method;
       pthread_mutex_lock( &driver->submit_lock );
 
-      if (driver->debug_output)
-        printf("LSF DRIVER submitting using method:%d \n",submit_method);
+      if (driver->debug_output){
+        printf("LSF DRIVER submitting using method:%d \n",submit_method); //TODO: remove?
+        res_log_add_fmt_message(LOG_INFO, NULL ,"LSF DRIVER submitting using method:%d \n",submit_method);
```

Comment:
The whole system with debug output in the lsf_driver was there because we did not have access to a logging facility - that is now changed. 

I would actually suggest completely removing the `driver->debug_output` facility.